Library/Formula/libhttpserver.rb

```diff
@@ -0,0 +1,49 @@
+class Libhttpserver < Formula
+  homepage "https://github.com/etr/libhttpserver"
+  url "https://github.com/etr/libhttpserver/archive/v0.7.1.tar.gz"
+  head "https://github.com/etr/libhttpserver.git"
+  sha1 "b36c8a5f523c04613d6a36c86bbb31a2b92aab83"
+
+  option "with-cpp11", "Enable c++11 std classes"
+  depends_on "libmicrohttpd"
+
+  depends_on "pkg-config" => :build
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  option :universal
+
+  patch :p1 do
+    url "https://github.com/etr/libhttpserver/pull/58.patch"
+    sha1 "20511129336d20585c8f9da6d0020f3ee5717bcc"
+  end
```

Comment:
Also, now it's merged let's just use `https://github.com/etr/libhttpserver/commit/3856cd.diff.